editor/plugins/node_3d_editor_plugin.h

```diff
@@ -632,6 +631,7 @@ class Node3DEditor : public VBoxContainer {
 
 	Node3DEditorViewportContainer *viewport_base = nullptr;
 	Node3DEditorViewport *viewports[VIEWPORTS_COUNT];
+	int last_used_viewport = 0;
```

Comment:
This looks out of place here. You could at least add some empty lines for separation.

Reply:
Above or below? I put it there because it's together with other viewport-related variables.

Reply:
It's integer among arrays/pointers. Just looks odd IMO, but it's purely visual, so if you ignore it it's fine too I guess.
The blank line should be below. Or you can move it entirely below this property block.

Reply:
The blank line has been added.